WikiWatershed/model-my-watershed
# Upgrade TR55

* Use latest version with updated CN
* Update the micro site model to match the data used by this version

Connects #1029 

To test:
- Run `collectstatic` and check that the microsite values for `developed_high` has changed compared to the current mmw-dev release
- Reprovision your worker and ensure you can still run a model.
